### ISSUE TITLE
Can you share the settings of Learning Rate?

### ISSUE THREAD

**Issue by ken-ouyang** [kentsyx/Neural-IMage-Assessment#2]
Hi, 

I have tried the default learning rate settings and it does not converge. Can you share the training settings? 

Many thanks.

**Comment by kentsyx** [comment]
Hi @ken-ouyang 

I used an lr of 0.001 for the conv base and 0.01 for the dense block, for which I got a 0.22 EMD loss. But I suspect they are not optimal and can be further improved.
